kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/DhcpClientLeaseBlock.java

```diff
@@ -0,0 +1,268 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates
```

Comment:
Remove initial date as this file is new.

Reply:
done